src/test/java/org/cactoos/text/StrictTest.java

```diff
@@ -75,13 +72,18 @@ public void returnsUnchangedIfPredicateIsPositive() {
      * @throws Exception If fails
      */
     @Test
-    public void failsIfNotMatchedWithPattern() throws Exception {
-        this.exception.expect(IllegalArgumentException.class);
-        this.exception.expectMessage("text");
-        new Strict(
-            Pattern.compile("^[a-zA-Z]+$"),
-            new TextOf("text12")
-        ).asString();
+    public void failsIfNotMatchedWithPattern() {
+        MatcherAssert.assertThat(
```

Comment:
@atapin please use `Assertion` instead of the static methods from `MatcherAssert`